content/en/_index.html

```diff
@@ -3,26 +3,138 @@
+		Read the Docs <i class="fas fa-arrow-alt-circle-right ml-2"></i>
 	</a>
 	<a class="btn btn-lg btn-light mr-3 mb-4" href="https://github.com/cdevents">
 		GitHub <i class="fab fa-github ml-2 "></i>
 	</a>
 </div>
+<div>
+	Embrace the freedom to pick and choose as you will, empower your builds with broad reusability and free your Continuous Delivery approach from vendor lock-in!
+<br/><br/>
+</div>
+<div>
+	With CDEvents, you have the freedom to adopt the Continuous Delivery approach that your product needs, with the flexibility to incorporate whatever tooling best suits your specific requirements.
+	<br/><br/>
+</div>
+<div>
+	CDEvents is a common specification for Continuous Delivery events, enabling interoperability in the complete software production ecosystem.
+	<br/>
+</div>
 <div class="mx-auto mt-5">
 	{{< blocks/link-down color="info" >}}
 </div>
 {{< /blocks/cover >}}
 
-{{< blocks/lead title="About CDEvents" color="dark">}}
+{{< blocks/lead title="About CDEvents" color="white">}}
+<p>
+	We are all too aware that teams building software products have a hidden overhead that they must manage. Everyone involved understands that you are building a machine to solve your customers' problem, and that this machine is an asset that must be invested in and maintained, in order to maximize its value. What many fewer people recognize is that to do this successfully, you must also own a second machine, whose only purpose is to build the first.
+</p>
+<div>
+	<p>The first machine, that is your product, is highly visible and commands all the attention and investment.</p>
+	<p>The second machine, that is critical to maintaining the value of the first, often sits in darkness, neglected and poorly understood, despite being essential to building all the value in your business.</p>
+</div>
+<p>
+	When you design your product, you have the freedom to choose from an almost infinite palette of reusable, modular libraries and frameworks. When you design your build system, however, your choices are much more limited, often being constrained to accepting the features offered by your current cloud vendor, or building something from first principles, which must be maintained with scant resources.
+</p>
+<p>
+	Although many tools exist in the CI/CD space, they have been unable to talk to each other in any standardized way, until today. The Continuous Delivery Foundation’s CDEvents project has been created to solve this interoperability problem, providing a new evolution in CI/CD capabilities.
+</p>
+{{< /blocks/lead >}}
+
+{{< blocks/lead title="CDF" color="light">}}
+CDEvents is an incubated project at the <a href="https://cd.foundation/projects">Continuous Delivery Foundation</a>
+
+<a href="https://cd.foundation/"><img src="https://raw.githubusercontent.com/cdfoundation/artwork/main/cdf/stacked/color/cdf-stacked-color.svg" width="20%"/></a>
+{{< /blocks/lead >}}
+
+{{< blocks/lead title="Benefits" color="white">}}
```

Comment:
This block sounds nice, but I think it's too much text for the home page, especially with the current formatting in a thick font back on white, centred on the page.

Some ideas could be to have tiles with some short text or logos which could be expanded to the full text, or links to separate pages with the full content.

Reply:
At this stage of maturity of the product, the landing page should be considered to be a sales pitch. It should be a 2-3 minute read of the scale of a news article, making a handful of clear points to attract attention. The point of doing this on the landing page is exactly because it is self-filtering. Interested parties will scroll down and those reacting with no spark to the idea will navigate away.

You need a narrative because the concept is too abstract for people to grasp easily without explanation.

The formatting is an example of the trade-off that has to be made when trying to optimise a site for desktop and mobile use. The text appears somewhat on the heavy side on a large display, but on a tablet or mobile device it is pitched at the density and word spacing of newsprint.

We should keep in mind that this is a sales pitch for a virtual product currently, so words are necessary.

Reply:
I do agree that a narrative could be very valuable here. But these paragraphs are maybe a bit unnecessarily long for people to not stop reading before they reach the last paragraph. I don't know how to abbreviate the text really, but an alternative could maybe be to change most of the text to be non-bold, and emphasizing the most important sub-sentences in bold instead? Kind of how Tekton does it on https://tekton.dev/

Reply:
A couple of important points here. The information is less than 1,000 words, making it around the same size as a typical new article or viral click-bait page. From a data-driven approach, readers who are engaged with a topic tend to consistently increase their engagement, up-to about 4,000 words, then behaviours diverge. See [Article length](https://whatsnewinpublishing.com/publishers-is-there-an-optimal-article-length/) for an example with more analysis.

Readers don't have to read the entire page, the links are always at the top, and there are jumping off points during the narrative. However, if the reader does not have sufficient context prior to getting to the specification, they will absolutely struggle to interpret those documents and will navigate away. The purpose of this front page in this context is to give the reader enough context to enable them to decide if it is worth investing in the commitment to read the specification, which is a much harder task. If we were using analytics, we could measure how many navigate away from the site without reading all the front page, and how many go on to the spec, having read some, or all of the front page.

Reply:
The second point is relating to accessibility. The font weights and heights used on this page are not my selection, they are calculated mathematically within Docsy, to align to the Google style guide. I believe that this is done from a mobile-first perspective because as someone with normal, age-related loss of close vision, I can comfortably read the text using these defaults on an iPhone, but have to strain to read the smaller text on the Tekton site. Docsy doesn't provide any out-of-the-box mechanism to rebalance font sizes on desktop displays vs mobile, so we would have to attempt to build and support that if we want to have optimised sizes across all devices.

There are some hacks that we could attempt in order to override the rendering, but this would impact the accessibility to those using screen readers, as well as the mobile audience. I believe that this needs a decision as to whether accessibility, mobile-access or desktop access is the priority for the design.

Reply:
I have extended Docsy to add a CDEvents-specific block type for the landing page text. This allows us to specify a lighter font weight. I have pushed this as far as I can because it is now starting to get hard to read on mobile.

I have also added an alternate set of styles for links, for dark backgrounds. Contrast ratios are better however I intend to revisit this in a later PR to dial in the final balance.

Reply:
Sounds good to me. Accepted.